### ISSUE TITLE
[Adaptive Card iOS - Table.FirstRowAsHeaders.json]: Voice Over is conveying the incorrect role as 'Heading' for the table headers.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8440]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Table.FirstRowAsHeaders.json.Jsons' and activate it.
7. The Card will open.
8. Navigate to the table header and observe the issue.

**Actual Result:**
While navigating to the table header, screen reader is conveying the incorrect role as 'Heading'. It is announcing as **'Header header heading text field double tap to edit'.**

**Expected Result:**
Voice Over should not convey the incorrect role for the table header. It should announce as **'Header Column 1 Row 1'** also it should announce the row and column information.

**Note:**
This issue is repro throughout the application having table.

**User Impact:**
It will impact the screen reader user as they will not be able to know whether they are navigating to the table or heading.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229165731-ed656b70-68f8-4464-a4d1-c4c56aaaf1ef.mov

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
closing this issue, the app wasn't built correctly perhaps, only cards under Scenario should be tested, not under Elements. I will make note of this and fix it.
